public/static/cql/mCODE.cql

```diff
@@ -4,9 +4,120 @@ using FHIR version '4.0.0'
 
 // CODESYSTEMS
 codesystem "SNOMEDCT": 'http://snomed.info/sct'
+codesystem "LOINC": 'http://loinc.org'
+codesystem "OBSCAT": 'http://terminology.hl7.org/CodeSystem/observation-category'
```

Comment:
It doesn't look like this has any uses and can be removed now that the code using it has been removed.

Reply:
Good catch. Removed